projectile.el

```diff
@@ -1124,6 +1124,12 @@ Controlled by `projectile-require-project-root'."
      (projectile-require-project-root (error "Projectile can't find a project definition in %s" dir))
      (t default-directory))))
 
+(defun projectile-acquire-root (&optional dir)
```

Comment:
I've meant to add a similar function when I added `projectile-ensure-project`, but I'm not sure about the name, as I don't think `acquire` is the best word we can use here.

Reply:
Again, you can propose a new name if you want. However, I would argue that "acquire" is fitting here, as it in my opinion has that "ensuring" character. From Merriam Webster (https://www.merriam-webster.com/dictionary/acquire), 1b: "to come into posession by unspecified means". I think this is fitting as you really always get the root (the user is prompted otherwise), and the unspecified means because there are two ways with which the root could be acquired, and due to the unspecified nature new ones could be added later perhaps?